server/src/main/java/io/littlehorse/common/model/corecommand/subcommand/TaskClaimEvent.java

```diff
@@ -106,6 +110,7 @@ public static TaskClaimEvent fromProto(TaskClaimEventPb proto, ExecutionContext
     public void initFrom(Message p, ExecutionContext context) {
         TaskClaimEventPb proto = (TaskClaimEventPb) p;
         taskRunId = LHSerializable.fromProto(proto.getTaskRunId(), TaskRunIdModel.class, context);
+        taskDefId = LHSerializable.fromProto(proto.getTaskDefId(), TaskDefIdModel.class, context);
```

Comment:
Do we need this? We already have the `TaskRunId`, so the server can get the `TaskRun` and find out the `TaskDefId` already.

This seems like extra stuff we are sending over the wire for no reason